addons/website/views/snippets.xml

```diff
@@ -365,6 +365,80 @@
     </section>
 </template>
 
+<template id="s_company_team" name="Company Team">
+    <section class="s_company_team">
+        <div class="container">
+            <div class="row mb32">
+                <h1 class="text-center">
+                    Meet the Executive Team
+                </h1>
+            </div>
```

Comment:
There can't be content in a `.row` which is not a `.col-*`. Here I think you should simply put the `h1` element as direct child of the `.container` (and put the `mb32` class on it if necessary).